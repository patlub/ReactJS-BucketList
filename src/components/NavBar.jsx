import { Link } from 'react-router-dom';
import React from 'react';

// styles
import '../App.css';

function NavBar() {
  return (
    <div className="row app-nav">
      <div className="col-md-8 col-sm-8 pull-left">
        <h3 id="app-title">BucketList</h3>
      </div>
      <div className="col-md-2 col-sm-2 nav-tab text-center">
        <Link to="logout">Logout</Link>
      </div>
    </div>
  );
}

export default NavBar;
