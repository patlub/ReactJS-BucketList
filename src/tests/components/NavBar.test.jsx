import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../../components/NavBar';
import { StaticRouter } from 'react-router';


it('Displays NavBar', () => {
  const rendered = renderer.create(
    <StaticRouter location="logout" context={''}>
      <NavBar />
    </StaticRouter>,
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});
