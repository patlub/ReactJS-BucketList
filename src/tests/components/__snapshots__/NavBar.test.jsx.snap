// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays NavBar 1`] = `
<div
  className="row app-nav"
>
  <div
    className="col-md-10 col-sm-10 pull-left"
  >
    <h3
      id="app-title"
    >
      BucketList
    </h3>
  </div>
  <div
    className="col-md-2 col-sm-2 nav-tab text-center"
  >
    <a
      href="/logout"
      onClick={[Function]}
    >
      Logout
    </a>
  </div>
</div>
`;
